### ISSUE TITLE
Unable to run test of ssgo

### ISSUE THREAD

**Issue by kt3k** [mdubourg001/ssgo#11]
When I tried to run the test of ssgo, I saw the following error:

```shellsession
$ vr test
error: Uncaught SyntaxError: The requested module './helpers.ts' does not provide an export named 'getTestEnv'
import { getTestEnv } from "./helpers.ts";
         ~~~~~~~~~~
    at <anonymous> (index.test.ts:17:10)
error: Failed at the test script
```

The file `src/test/helpers.ts` looks empty, and it seems deleted in [this commit](https://github.com/mdubourg001/ssgo/commit/c4a7a9e1803655f6cc96ebff71a391c76543405f). Is this intentional state?

**Comment by mdubourg001** [comment]
Hi, thanks for reporting this. 

I think you forgot to commit the changes you made on the `src/test/helpers.ts` file in your MR #13. No problem about that tho, everything should be fixed in the last release (v0.19.2).

**Comment by kt3k** [comment]
Hey, thanks for taking care of this issue. Now I can run `vr test` on master branch 👍 Thanks!
